lib/streamlit/watcher/file_watcher.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 from typing import Optional, Union, Type, Callable
+import click
```

Comment:
nit: should be a newline here to separate stdlib and third party library imports

Reply:
updated